dotcom-rendering/playwright/lib/load-page.ts

```diff
@@ -1,9 +1,131 @@
-import type { Page } from '@playwright/test';
-import { PORT } from 'playwright.config';
+import type { Cookie, Page } from '@playwright/test';
+import { ORIGIN } from '../../playwright.config';
 import type { FEArticle } from '../../src/frontend/feArticle';
-import { validateAsFEArticle } from '../../src/model/validate';
+import type { FEFront } from '../../src/frontend/feFront';
+import {
+	validateAsFEArticle,
+	validateAsFEFront,
+} from '../../src/model/validate';
 
-const BASE_URL = `http://localhost:${PORT}`;
+type LoadPageOptions = {
+	queryParams?: Record<string, string>;
+	queryParamsOn?: boolean;
+	fragment?: `#${string}`;
+	waitUntil?: 'domcontentloaded' | 'load';
+	region?: 'GB' | 'US' | 'AU' | 'INT';
+	preventSupportBanner?: boolean;
+	overrides?: {
+		configOverrides?: Record<string, unknown>;
+		switchOverrides?: Record<string, unknown>;
+		feFixture?: FEArticle | FEFront;
+	};
+};
+
+type LoadPageParams = {
+	page: Page;
+	path: string;
+} & LoadPageOptions;
+
+/**
+ * @param path The path for a DCR endpoint path
+ *		e.g. `/Article/https://www.theguardian.com/world/2025/aug/19/the-big-church-move-sweden-kiruna-kyrka`
+ * @returns The Frontend URL to fetch the JSON payload
+ *		e.g. `https://www.theguardian.com/world/2025/aug/19/the-big-church-move-sweden-kiruna-kyrka.json`
+ */
+const getFrontendJsonUrl = (path: string) => {
+	const secondSlashIndex = path.indexOf('/', 1);
+	const contentUrl = path.substring(secondSlashIndex + 1);
+	return `${contentUrl}.json`;
+};
+
+/**
+ * @param path The Frontend URL to fetch the JSON payload
+ *		e.g. `https://www.theguardian.com/world/2025/aug/19/the-big-church-move-sweden-kiruna-kyrka`
```

Comment:
Does this actually take a DCAR path, i.e. one with an `/Article` or `/Front` prefix?